unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/Definitions/LoaderDefinitions/ILoader.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using Cysharp.Threading.Tasks;
+using UnityEngine;
+
+namespace AvatarSystem
+{
+    public interface ILoader : IDisposable
+    {
+        public enum Status
+        {
+            Idle,
+            Loading,
+            Succeeded,
+            Failed_Minor,
+            Failed_Mayor
```

Comment:
```suggestion
            Failed_Major
```
In english is `Major`, not `Mayor`.

Reply:
🤦